lib/improver/uv_index.py

```diff
@@ -93,5 +93,7 @@ def calculate_uv_index(uv_upward, uv_downward, scale_factor=3.6):
         uv_index.data = (uv_upward.data +
                          uv_downward.data) * scale_factor
         uv_index.standard_name = "ultraviolet_index"
+        uv_index.long_name = None
+        uv_index.var_name = None
```

Comment:
I believe these three lines can be dropped (95, 96, 97) and replaced with:
```
uv_index.rename("ultraviolet_index")
```

Reply:
Yes that works, thank you. I have changed it now.